src/routes/+page.svelte

```diff
@@ -71,6 +73,9 @@
 				break;
 			}
 		}
+		const md = markdownit('commonmark');
+		history[history.length - 1].content = md.render(history[history.length - 1].content);
+		history[history.length - 1].content = sanitizeHtml(history[history.length - 1].content)
```

Comment:
I think these might need to be flipped, because the md render should convert it to html while the sanitizeHtml will get rid of all the formatted html. So we may want to remove all html then render the markdown part instead.